tests/functional/provisioning/test_dev_path_mapping_file.py

```diff
@@ -792,3 +792,93 @@ def test_dev_path_mapping_heketi_node_delete(self):
             use_percent, use_percent_after,
             "Failed to execute IO's in the app pod {} after respin".format(
                 pod_name))
+
+    @pytest.mark.tier4
+    @podcmd.GlustoPod()
+    def test_dev_path_volume_expansion(self):
+        """Validate dev path mapping for file volumes expansion"""
+
+        pvc_size, pvc_amount, expand_pvc_size = 1, 2, 2
+        vol_names, pvc_names = [], []
+
+        # Create PVC's
```

Comment:
```suggestion
        # Create storage class with expansion option and PVCs
```
Don't consider writing apostrophe for PVC